renting-leasing-expenses-bot/README.md

```
@@ -1 +1,110 @@
+* `pip >= 24.2`
+
+## Instalación
+
+Una vez descargado el repositorio debemos de crear un [ambiente virtual](https://docs.python.org/3/library/venv.html) para poder instalar las dependencias
+a través del manejador de paquetes `pip`.
+
+Una vez tengamos el repositorio y nuestro ambiente virtual debemos de seguir el 
+siguiente comando:
+
+```bash
+$ pip install -r requirements.txt
```

Markdownlint (0.37.0)

Dollar signs used before commands without showing output

(MD014, commands-show-output)